core/CliMulti/Process.php

```diff
@@ -129,6 +134,7 @@ private function pidFileSizeIsNormal()
 
     public function finishProcess()
     {
+        $this->finished = true;
```

Comment:
technically, once finished, we wouldn't need to call `deleteFileIfExists` again but we're ignoring this for now just in case for some reason the file was not correctly removed. Could add this check though. `finishProcess` could be called twice for example during regular job and when `climulti:cleanup()` is called.